node/tools/src/config.rs

```diff
@@ -194,6 +221,90 @@ impl<'a> ConfigPaths<'a> {
     }
 }
 
+impl AppConfig {
+    pub fn default_for(validators_amount: usize) -> (AppConfig, Vec<validator::SecretKey>) {
+        // Generate the keys for all the replicas.
+        let rng = &mut rand::thread_rng();
+
+        let mut genesis = validator::GenesisSetup::empty(rng, validators_amount);
+        genesis
+            .next_block()
+            .payload(validator::Payload(vec![]))
+            .push();
+        let validator_keys = genesis.keys.clone();
+
+        (
+            Self {
+                server_addr: SocketAddr::new(std::net::Ipv4Addr::UNSPECIFIED.into(), NODES_PORT),
```

Comment:
either consistently import the types from std::net, or consistently use the qualified form, or import `std::net`.